yarnitor/api.py

```diff
@@ -19,7 +19,11 @@ def get_model():
 
 @api_bp.route('/api/status')
 def status():
-    return jsonify({'status': 'ok'})
+    ym = get_model()
```

Comment:
Why are we importing and then instantiating a yarn model from within a function? This feels like some sort of hack to get around something.